setup.cfg

```
@@ -38,7 +38,7 @@ install_requires =
     eciespy>=0.3.13; python_version>="3.11"
     typing_extensions
     typer
-    aleph-message==0.4.1
+    aleph-message~=0.4.2
```

```suggestion
    aleph-message~=0.4.1
```
This should be like that to accept `0.4.1` and `0.4.2` versions